http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -74,24 +75,6 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
   private def getListenerReleaseName(appReleaseName: Release): Release =
     Release(s"${appReleaseName.asString}-listener-rls")
 
-  private def retrieveWsmReferenceDatabases(resourceApi: ResourceApi,
```

Comment:
Just moved this method down in the file, next to the other private DB-related methods.